### ISSUE TITLE
Not possible to send NEO GAS from wallet

### ISSUE THREAD

**Issue by matthieuvandiepen** [neo-project/neo#1250]
Hello, are you aware in the moment it is not possible to send GAS from ANY wallet?

I tried different wallets, but with none of them it works.

Sending NEO is working OK, sending GAS doesn't result in a transaction on the blockchain.

I tried this with 4 different wallets, no succes. So this seems to be a problem in de NEO infrastructure.

**Comment by lock9** [comment]
Hello,
I found a GAS transaction 4 minutes ago: 
https://neoscan.io/transaction/ee92e376902b543a7e3e206a7b32fcbbd9885ae8f4610fdc93f9f1c4f874fd15

Can you provide more details? Do you get any error messages? 
How often do you claim gas? There is an issue if you have too many small claims. Do you think that can be the issue?

**Comment by matthieuvandiepen** [comment]
Hi, thx!

messages: it depends on the wallet I use. The most information I get from the neotracker.io wallet:
Transfer initiated to send 8.0499 GAS to address:
AK3NLxuBEUHgdYLCri5nxzPPSKk2kw2FPT

Transactions typically take up to one minute to confirm and appear on the explorer:

5a23cc6067a935bc56ec8ffaf5d9d3c0731a33c7935fb7250db72ab4b9def2e0

But then it never arrives on the blockchain.

I don;t claim gas often, I just claimed it because I wanted to remove the gas to another wallet.

I also saw a mention of a simmular issue here:
https://github.com/CityOfZion/neon-wallet/issues/1881

thanks again

**Comment by lock9** [comment]
Hello @matthieuvandiepen, are you adding a fee? Wallets usually support adding a fee to the transaction.

**Comment by TechMagy** [comment]
Hello Team!

I've encountered what sounds like a similar issue with the Neon Wallet (v2.4.0) which sends NEO fine, however, GAS won't send. This is with all fee options tested. 

Test 1 - sending 10 GAS from NEON to Binance without a fee
![Test 1](https://user-images.githubusercontent.com/38993528/73581611-d9289c80-44dd-11ea-9827-8bf58284f46f.JPG)

Test 2 - sending 10 GAS from NEON to Binance with a 0.001 GAS fee (Fast)
![Test 2](https://user-images.githubusercontent.com/38993528/73581688-2a389080-44de-11ea-87d3-0969f1fd220c.JPG)

Test 3 - sending 10 GAS from NEON to Binance with a 0.05 GAS fee (Faster)
![Test 3](https://user-images.githubusercontent.com/38993528/73581775-84d1ec80-44de-11ea-943c-1a728fb2f5dd.JPG)

Test 4 - sending 10 GAS from NEON to Binance with a 0.1 GAS fee (Fastest)
![Test 4](https://user-images.githubusercontent.com/38993528/73581879-db3f2b00-44de-11ea-903e-458b8972021a.JPG)

In all cases there is the yellow banner to sign the transaction in the Neon Wallet (on Ledger Nano S with current firmware - v 1.6.0) which disappears before a message is shown on the Ledger device. This is the replaced by the error messages shown above.

The highest GAS amount that can be selected in the Neon Wallet is 0.1 GAS but the error is claiming that 0.18687 GAS is required. How do we overcome this blockage?

No records of attempted transactions show in the Activity section of the Neon Wallet. I've changed from 'automatic' network selection to manually selecting nodes.

On my Ledger Nano S I currently have ETH, NEO and BTC apps installed.
